ale_linters/python/pylint.vim

```diff
@@ -0,0 +1,9 @@
+" Author: keith <k@keith.so>
+" Description: pylint for python files
+
+call ale#linter#Define('python', {
+\   'name': 'pylint',
+\   'executable': 'pylint',
+\   'command': 'pylint --output-format text --msg-template="{path}:{line}:{column}: {msg_id} {msg}" --reports n %s',
```

Comment:
I think there's a problem here. I think you're formatting in the filename, which means that the linter is running on the version of the file on disk, but not on the contents of the current buffer. You should try using `ale#util#stdin_wrapper`, which is used in some other linters.